### ISSUE TITLE
The highscore server is down

### ISSUE THREAD

**Issue by derek1906** [benedictfritz/Avalanche#1]
The highscore server is down :(

**Comment by benedictfritz** [comment]
@derek1906 Yeah, sorry about that. This project is quite old and was using Google AppEngine or some other thing that is now deprecated to run the highscore board. I might get around to getting a new one up eventually, but I wouldn't count on it. Apologies.

**Comment by derek1906** [comment]
That's ok, just happened to visit this page and decided to leave a nice message.
